roles/shibd/defaults/main.yml

```diff
@@ -3,6 +3,8 @@
 apache: apache2
 # use httpd for CentOS 7
 callback_url: /users/auth/shibboleth/callback
+idp_check_Address: true
```

Comment:
I'd set the default to "false," and add a comment letting people know why - that if you set it to "true" you get better security but you may get Shibboleth errors when folks log in from a VPN.